### PR TITLE
Added Class and Employee CRUD classes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: crud/ClassCRUD/src/ClassCRUD.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+public class ClassCRUD {
+
+	public void insertClass(Statement st, String class_id, String stream_id, String user_id){
+		try {
+			System.out.println("Inserting a class...");
+			int rows = st.executeUpdate("INSERT INTO Student_Performance.Class VALUES ('" + class_id + "','" + stream_id + "','" + user_id  + "')");
```
Comment (kabollinger03) [inline crud/ClassCRUD/src/ClassCRUD.java:11]: https://xkcd.com/327/

SQL Injection is serious. Should at the very least use a prepared statement. Can still be modular that way, but not vulnerable. Prepared statement can be written entirely inside Java.

**Review by kabollinger03 — DISMISSED** [review]

**Comment by apennella** [comment]
Im not seeing any problems with it.

**Comment by kabollinger03** [comment]
> Im not seeing any problems with it.

Look at it tomorrow... But did you read the comic? What happens if I pass the paramater `user_id` as 
`42"); DROP TABLE student_performance; --`

I don't think `executeUpdate` checks for that.

File: crud/ClassCRUD/src/ClassCRUD.java (hunk 1 above)
Comment (kabollinger03) [reply crud/ClassCRUD/src/ClassCRUD.java:11]: Use prepared statement or simmilar: https://www.javacodegeeks.com/2012/11/sql-injection-in-java-application.html

Seriously, use prepared statement WITH `?` PARAMATERS https://vladmihalcea.com/a-beginners-guide-to-sql-injection-and-how-you-should-prevent-it/

Comment (kabollinger03) [reply crud/ClassCRUD/src/ClassCRUD.java:11]: I notice it says Oracle will reject it.... but it is still terrible practice, and if you do that on MySQL you are terrible vulnerable.

**Review by ChrisSMendoza — APPROVED** [review]

**Review by apj11c — APPROVED** [review]

**Review by apennella — APPROVED** [review]
👍 🥇

**Review by cgreen9008 — COMMENTED** [review]
Just out of curiosity, do these need to be separate classes? I was under the impression methods on their own would work.

**Comment by kabollinger03** [comment]
> "too much re-work at this point"

Ok. Sure. Just know that once we have a cohesive product I will dedicate days to hitting this with every SQL Injection trick in the book.

**Pull request merged**